Makefile

```diff
@@ -0,0 +1,63 @@
+FLAGS =
+TESTENVVAR =
+REGISTRY = quay.io/redhat
```

Comment:
Can we already push to this registry, or is this a placeholder?

Reply:
We already can do this, I have the permission now.